python-sdk/tests/benchmark/synthetic_csv_generator.py

```diff
@@ -34,4 +33,16 @@ def generate_data(row_size, file_name):
     file_obj.close()
 
 
-generate_data(11000000, "one_gb.csv")
+generate_data(125, "/fake_dataset/csv/ten_kb.csv")
+generate_data(1100, "/fake_dataset/csv/hundred_kb.csv")
+generate_data(107430, "/fake_dataset/csv/ten_mb.csv")
+generate_data(1074300, "/fake_dataset/csv/hundred_mb.csv")
+generate_data(11000000, "/fake_dataset/csv/one_gb.csv")
+generate_data(11000000 * 2, "/fake_dataset/csv/two_gb.csv")
+generate_data(11000000 * 5, "/fake_dataset/csv/five_gb.csv")
+generate_data(11000000 * 2, "/fake_dataset/csv/five_gb/second.csv")
+generate_data(11000000 * 2, "/fake_dataset/csv/five_gb/first.csv")
+generate_data(11000000, "/fake_dataset/csv/five_gb/third.csv")
+generate_data(11000000 * 2, "/fake_dataset/csv/ten_gb/second.csv")
+generate_data(11000000 * 2, "/fake_dataset/csv/ten_gb/first.csv")
+generate_data(11000000, "/fake_dataset/csv/ten_gb/third.csv")
```

Comment:
@sunank200 this does not equal ten gb, or am I missing something? 🤔

Reply:
@feluelle ten_gb folder in this script contains two gb files(2 of them) and a one gb file. These files are again copied twice with different names on the VM they are created to make it ten gb.